Simplenote/Classes/FileManager+Simplenote.swift

```diff
@@ -14,19 +14,18 @@ extension FileManager {
 
         return url
     }
-
-    /// Writes a given Note to the Documents folder
+    
+    /// User's Temporary Directory
     ///
-    class func writeToDocuments(note: Note) -> URL? {
-        guard let payload = note.content else {
-            return nil
-        }
-
-        let filename = String(format: "%@.txt", note.simperiumKey)
-        let targetURL = FileManager.documentsURL.appendingPathComponent(filename)
-
+    class var temporaryDirectoryURL: URL {
+        return FileManager.default.temporaryDirectory
+    }
+    
+    /// Writes a given String to the documents folder
+    ///
+    class func writeStringToURL(string: String, to targetURL: URL) -> URL? {
```

Comment:
This API doesn't use `FileManager` (and it was originally misplaced).

It does interact with a String + URL, how about if we move this one to the spot where it's actually used? (`SimplenoteActivityItemSource`)